docs/content/bootstrapper.md

```diff
@@ -146,3 +146,36 @@ A few default settings are applied:
 * The bootstrapper is silenced and only errors are displayed. `-v` can be used
   once to restore normal output or twice to show more details.
 * If no version is specified a default `--max-file-age` of `12` hours is used.
+
+### Setting version in magic mode
+
+There are a few ways to set the version requested in magic mode. 
```

Comment:
This section seem to summarize what's already in the file : why ?

Also :
* It's missing the nicest way to do that : [add `version x.y.z` to the `paket.dependencies` file](https://github.com/fsprojects/Paket/blob/master/docs/content/bootstrapper.md#in-paketdependencies)
* The existing documentation on [Environment variables](https://github.com/fsprojects/Paket/blob/master/docs/content/bootstrapper.md#with-environment-variables) isn't changed with the new name (Keeping compat is nice but I don't think the non-posix version need to be documented).

Reply:
Seemed non-obvious for magic mode to me.  Maybe I should trim it to just mentioning `paket.exe.config` ?

Reply:
Think it's worth moving the how to do env var commands to under the env var heading?

Reply:
👍 on latest, maybe specifying in the **Setting version in magic mode** section that all methods (except for the command line one) works in this mode (with the note about renaming). But it can be changed later.